generator/src/main/java/com/faforever/neroxis/generator/Biome.java

```diff
@@ -0,0 +1,32 @@
+package com.faforever.neroxis.generator;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public enum Biome {
```

Comment:
This name needs to be different otherwise it is confusing having two classes named Biome